src/Core/Models/Table/Cipher.cs

```diff
@@ -21,6 +21,7 @@ public class Cipher : ITableObject<Guid>, ICloneable
         public DateTime CreationDate { get; internal set; } = DateTime.UtcNow;
         public DateTime RevisionDate { get; internal set; } = DateTime.UtcNow;
         public DateTime? DeletedDate { get; internal set; }
+        public Enums.CipherRepromptType Reprompt { get; set; }
```

Comment:
Is the intent that `None`, represented by `0` (zero) be returned as default, or stored as default, to the `NOT NULL` column, `Reprompt` on the `dbo.Cipher` table? 

I would rather see that column made nullable, so we instead just want to use a `null` value for this with `public Enums.CipherRepromptType? Reprompt { get; set; }` (implementing instead `Nullable<Enums.CipherRepromptType>`) with a default value of `null` instead of `0` (takes less storage in SQL Server)

Reply:
I mostly tried to follow how we handle enums in the other tables. I don't have anything against making the field nullable and treading null as 0.

Reply:
Changed to be nullable, the API handles giving it a default value in the response model, and sets it to null in the request handler (should it be None).